client/state/sites/selectors.js

```diff
@@ -58,6 +59,22 @@ export function isSiteConflicting( state, siteId ) {
 }
 
 /**
+ * Returns true if site is a Jetpack site.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether site is a Jetpack site
+ */
+export function isJetpackSite( state, siteId ) {
+	const site = getSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+
+	return get( site, 'jetpack' );
```

Comment:
We can assume `site` is truthy at this point, given the above condition, so `get` seems unnecessary in lieu of `return site.jetpack;` . Alternatively, you could use `get` but specify a [default value](https://lodash.com/docs#get) of `null` for when the site doesn't exist:

```js
return get( getSite( state, siteId ), 'jetpack', null );
```

While certainly more concise, I'm on the fence about whether it's a bit harder to follow.